src/components/Auth/Auth.tsx

```diff
@@ -0,0 +1,99 @@
+import { h, FunctionComponent } from 'preact'
+import { AuthCheckProcessor } from './AuthCheckProcessor'
+import { FaceTecSDK } from '~auth-sdk/FaceTecSDK.js/FaceTecSDK'
+import { Config } from './AuthConfig'
+import { FaceTecStrings } from './assets/FaceTecStrings'
+import type { WithLocalisedProps } from '~types/hocs'
+import type { StepComponentBaseProps } from '~types/routers'
+import { useEffect, useState } from 'preact/hooks'
+
+type Props = StepComponentBaseProps & WithLocalisedProps
+
+type AuthConfigType = {
+  token: string
+  production_key_text: string
+  device_key_identifier: string
+  public_key: string
+}
+
+const AuthCapture: FunctionComponent<Props> = ({
+  token,
+  nextStep,
+  translate,
+  events,
+  customUI,
+}) => {
+  const [authConfig, setAuthConfig] = useState<AuthConfigType>({
+    token: '',
+    production_key_text: '',
+    device_key_identifier: '',
+    public_key: '',
+  })
+  const [sessionInit, setSessionInit] = useState(false)
+
+  useEffect(() => {
+    if (FaceTecSDK.getStatus() === 0 && !sessionInit) {
+      FaceTecSDK.setCustomization(
+        Config.getAuthCustomization(false, customUI || {})
+      )
+      FaceTecSDK.setDynamicDimmingCustomization(
+        Config.getAuthCustomization(true, customUI || {})
+      )
+      if (authConfig.token && !sessionInit) initFaceTec()
+      else getConfig()
+    } else onLivenessCheckPressed()
+  })
+
+  const initFaceTec = () => {
+    const authAlias = '../../../../auth-sdk/FaceTec/'
```

Comment:
I'm a bit uncomfortable with these relative paths, but given that FaceTec has a different resolver for the paths, I guess we have to compromise with this. Let's take an eye on this thing in the future.